go-controller/pkg/cni/helper_linux_test.go

```diff
@@ -318,6 +318,30 @@ func TestSetupNetwork(t *testing.T) {
 				{OnCallMethodName: "Attrs", OnCallMethodArgType: []string{}, RetArgList: []interface{}{&netlink.LinkAttrs{Name: "testIfaceName", Flags: net.FlagUp}}},
 			},
 		},
+		{
+			desc:    "test skip ip config",
+			inpLink: mockLink,
+			inpPodIfaceInfo: &PodInterfaceInfo{
+				SkipIPConfig: true,
+				PodAnnotation: util.PodAnnotation{
+					IPs:      ovntest.MustParseIPNets("192.168.0.5/24"),
+					MAC:      ovntest.MustParseMAC("0A:58:FD:98:00:01"),
+					Gateways: ovntest.MustParseIPs("192.168.0.1"),
+					Routes: []util.PodRoute{
+						{
+							Dest:    ovntest.MustParseIPNet("192.168.1.0/24"),
+							NextHop: net.ParseIP("192.168.1.1"),
+						},
+					},
+				},
```

Comment:
Can we avoid setting the IP annotation ? 

Just trying to understand if the test input couldn't be simpler / more focused.

Reply:
I want to test it how is going to be received by CNI that's why the ip config is added even if it's going to be skip.